src/Source/DoctrineDBALConnectionSourceIterator.php

```diff
@@ -80,13 +80,11 @@ public function rewind(): void
     {
         $statement = $this->connection->prepare($this->query);
 
-        // TODO: Keep only the if part when dropping support for Doctrine DBAL < 3.1
+        // TODO: Keep only the else part when dropping support for Doctrine DBAL < 3.1
```

Comment:
The if part is the good one

Reply:
I need to check tomorrox but I thought I had dbal 3.2 and no executeQuery method

Reply:
https://github.com/sonata-project/exporter/pull/562/files#r772146858

I think it's the else part we need to keep.
I have dbal 3.2 locally and no executeQuery method